strongbox-user-management/src/main/java/org/carlspring/strongbox/authentication/api/impl/StrongboxBuiltinAuthenticator.java

```diff
@@ -0,0 +1,43 @@
+package org.carlspring.strongbox.authentication.api.impl;
+
+import org.carlspring.strongbox.authentication.api.AuthenticationSupplier;
+import org.carlspring.strongbox.authentication.api.Authenticator;
+import org.carlspring.strongbox.authentication.api.impl.userdetails.StrongboxUserDetailService;
+
+import javax.annotation.Nonnull;
+import javax.inject.Inject;
+
+import org.springframework.security.authentication.AuthenticationProvider;
+import org.springframework.security.authentication.dao.DaoAuthenticationProvider;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.stereotype.Component;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+@Component
+public class StrongboxBuiltinAuthenticator
+        implements Authenticator
+{
+
+    @Inject
+    @StrongboxUserDetailService.StrongboxUserDetailServiceQ
```

Comment:
Also, I don't think we need to prefix everything with `Strongbox`... Can't we just call it `UserDetailService`?